src/components/Navbar.js

```diff
@@ -12,14 +12,14 @@ export const Navbar = () => (
     <Link to="/disciplines" activeClassName="selected">
       Disciplines
     </Link>
-    <Link to="/news" activeClassName="selected">
-      News
+    <Link to="/timetable" activeClassName="selected">
+      Timetable
     </Link>
     <Link to="/team" activeClassName="selected">
-      Meet our team
+      Meet the team
```

Comment:
Thanks for your help.
Can you just leave the text saying `Meet our team`. Thanks